net/adblock/files/adblock.sh

```diff
@@ -604,7 +604,7 @@ f_list()
 				out_rc="${?}"
 				if [ "${out_rc}" -eq 0 ]
 				then
-					adb_whitelist_rset="/^([[:alnum:]_-]+\\.)+[[:alpha:]]+([[:space:]]|$)/{gsub(\"\\\\.\",\"\\\\.\",\$1);print tolower(\"^\"\$1\"\\\\|\\\\.\"\$1)}"
+					adb_whitelist_rset="/^([[:alnum:]_-]+\\.)+[[:alpha:]]+([[:space:]]|$)/{gsub(\"\\\\.\",\"\\\\.\",\$1);print tolower(\"^(|.*\\\\.)\"\$1\"$\")}"
```

Comment:
Is it possible to have spaces after the domain name and the end of line? Maybe `[[:space:]]$` should be used here?

Reply:
Maybe I don't understand you correctly: you mean a space after line end? The current regex covers a space or a line end after the domain name ... IMHO sufficient.

Reply:
`^(|.*\.)api\.segment\.io$` will exclude `blah.api.segment.io`, but if you add spaces or tabs after `io`. it will not be excluded.

Reply:
I am not familiar with the code that well, but if you `trim` each domain from black and whitelist, then this should not happen. That way black/white-lists with spaces/tabs before and after the name will be handled and this will decrease the file sizes in that case.

Reply:
The input file (raw.whitelist) does not have any spaces, therefore your test case is not valid.

Reply:
But the regex is applied to the black list: can it have whitespaces?

Reply:
No, both lists are pre-filtered and trimmed.